packages/tasks/src/model-libraries.ts

```diff
@@ -686,6 +686,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	"ssr-speech": {
+		prettyLabel: "SSR-Speech",
+		repoName: "SSR-Speech",
+		repoUrl: "https://github.com/WangHelin1997/SSR-Speech",
+		filter: false,
+		countDownloads: `path_extension:".pth" AND path_extension:".th"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:".pth"`,
```
Let's count the downloads only on 1 file per repo (here English.pth and Mandarin.pth). Since both the .pth and the .th files seem to be needed to load the model, counting downloads on both would count everything double